views/marketplace/index.php

```diff
@@ -19,7 +19,7 @@
             <?php if (count($products) === 0): ?>
                 <div class="panel">
```

Comment:
Shouldn't matter much but `<div class="panel panel-default">` could also be used here. :thinking:

Reply:
yep